SPECS/fakeroot/fakeroot.spec

```diff
@@ -2,39 +2,39 @@
 
 Summary:        Gives a fake root environment
 Name:           fakeroot
-Version:        1.25.3
-Release:        2%{?dist}
+Version:        1.28
+Release:        1%{?dist}
 # setenv.c: LGPLv2+
+# contrib/Fakeroot-Stat-1.8.8: Perl (GPL+ or Artistic)
 # the rest: GPLv3+
-License:        GPLv3+ AND LGPLv2+
+License:        GPLv3+ AND LGPLv2+ AND (GPL+ OR Artistic)
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://tracker.debian.org/pkg/fakeroot
 Source0:        https://cdn-aws.deb.debian.org/debian/pool/main/f/fakeroot/%{name}_%{version}.orig.tar.gz
 # Debian package patches, from debian.tar.xz
-Patch0:         debian_eglibc-fts-without-LFS.patch
 Patch2:         debian_fix-shell-in-fakeroot.patch
 # Address some POSIX-types related problems.
 Patch4:         fakeroot-inttypes.patch
 # Fix LD_LIBRARY_PATH for multilib: https://bugzilla.redhat.com/show_bug.cgi?id=1241527
 Patch5:         fakeroot-multilib.patch
-# https://bugzilla.redhat.com/show_bug.cgi?id=1889862
-Patch6:         glibc_2.33_fix.patch
 Patch7:         relax_tartest.patch
```

Comment:
NIT: Since we are moving to `1.28` can we drop this patch and try the [fix](https://salsa.debian.org/clint/fakeroot/-/merge_requests/14) that the upstream community has proposed.
Hopefully it works :)

Reply:
I think this PR already updates `relax_tartest.patch` to exactly that fix from your link.

Reply:
Ahh missed it as I did not see a changelog entry :(
my bad